examples/electron/src-gen/backend/main.js

```diff
@@ -29,15 +29,16 @@ function start() {
     application.use(express.static(path.join(__dirname, '../../lib'), {
         index: 'index.html'
     }));
-    application.start();
+
+    module.exports.default = application.start(false, 0)
 }
 
 Promise.resolve()
```

Comment:
The promise that would get resolved is the one that's returned by `application.start()`? If that is the case, do we not call `resolve` in the callback?